modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -71,39 +71,42 @@ Note the catalog for your desired Operator.
 $ oc describe packagemanifests <operator_name> -n openshift-marketplace
 ----
 
-. An Operator group, defined by an `OperatorGroup` object, selects target namespaces in which to generate required RBAC access for all Operators in the same namespace as the Operator group.
+. If the Operator you intend to install uses the `AllNamespaces` install mode, skip this step, because the `openshift-operators` namespace already has an appropriate Operator group in place by default, called `global-operators`.
 +
-The namespace to which you subscribe the Operator must have an Operator group that matches the install mode of the Operator, either the `AllNamespaces` or `SingleNamespace` mode. If the Operator you intend to install uses the `AllNamespaces` mode, the `openshift-operators` namespace already has the appropriate `global-operators` Operator group in place.
+If the Operator you intend to install uses the `SingleNamespace` install  mode and you do not already have an appropriate Operator group in place, you must create one by following these steps:
 +
-However, if the Operator uses the `SingleNamespace` mode and you do not already have an appropriate Operator group in place, you must create one.
+--
+.. An Operator group, defined by an `OperatorGroup` object, selects target namespaces in which to generate required RBAC access for all Operators in the same namespace as the Operator group.
 +
-[NOTE]
-====
-* The web console version of this procedure handles the creation of the `OperatorGroup` and `Subscription` objects automatically behind the scenes for you when choosing `SingleNamespace` mode.
-* You can only have one Operator group per namespace. For more information, see "Operator groups".
-====
-
-.. Create an `OperatorGroup` object YAML file, for example `operatorgroup.yaml`:
+Create an `OperatorGroup` object YAML file, for example `operatorgroup.yaml`, for `SingleNamespace` install mode:
 +
-.Example `OperatorGroup` object
+.Example `OperatorGroup` object for `SingleNamespace` install mode
 [source,yaml]
 ----
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
   name: <operatorgroup_name>
-  namespace: <namespace>
+  namespace: <namespace> <1>
 spec:
   targetNamespaces:
-  - <namespace>
+  - <namespace> <1>
 ----
+<1> For `SingleNamespace` install mode, use the same `<namespace>` value for both the `metadata.namespace` and `spec.targetNamespaces` fields.
 
 .. Create the `OperatorGroup` object:
 +
 [source,terminal]
 ----
 $ oc apply -f operatorgroup.yaml
 ----
+--
++
+[NOTE]
+====
+* The web console version of this procedure handles the creation of the `OperatorGroup` and `Subscription` objects automatically in the background for you when choosing `SingleNamespace` mode.
```

Comment:
Not to be flippant, but why do I care what the web console version of this procedure does? Seems as though this would be better in the web console procedures.

Reply:
I think it was initially added because we primarily recommend users go with the web console method since this Operator group stuff can get tricky when manually creating.

Reply:
Moved this to the intro and gave it more context wrt preferred method.